backend/src/main/scala/com/softwaremill/bootzooka/common/Utils.scala

```diff
@@ -1,12 +1,12 @@
 package com.softwaremill.bootzooka.common
 
-import java.util.Random
+import java.security.SecureRandom
 
 object Utils {
 
   def randomString(length: Int) = {
     val sb = new StringBuffer()
-    val r  = new Random()
+    val r  = new SecureRandom()
```

Comment:
:+1:

Reply:
it doesn't matter that we create a new instance each time?

Reply:
It might, moved it out of the method